events/wc_frostwolf_events.txt

```diff
@@ -102,63 +141,63 @@ province_event = {
 narrative_event = {
 	id = WCFW.10
 	title = EVTTITLE_WCFW_10
-	desc = EVTDESC_WCFW.10 #These lands remind us of the home, but they seems to be inhabited.
+	desc = EVTDESC_WCFW_10 # We went North, now what...
 	picture = GFX_evt_khazmodanroad
 	border = GFX_event_narrative_frame_war
 
 	is_triggered_only = yes
 
 	option = {
-		name = EVTOPTA_WCFW.10
+		name = EVTOPTA_WCFW_10 # We will continue north. (Alterac)
 		custom_tooltip = { text = EVTOPTA_WCFW_10_tooltip }
-		#Go Northern
+		# Go North
 		73 = {
 			province_event = {
-				id = WCFW.111 #Northern
+				id = WCFW.111 # North
 				days = 31
 			}
 		}
 	}
 	option = {
-		name = EVTOPTB_WCFW.10
+		name = EVTOPTB_WCFW_10 # We will go towards the west. (Ironforge)
 		custom_tooltip = { text = EVTOPTB_WCFW_10_tooltip }
-		#Go Western
+		# Go Western
 		73 = {
 			province_event = {
-				id = WCFW.112 #Western
+				id = WCFW.130 # West
 				days = 31
 			}
 		}
 	}
-	option = { # I stupid orc I no understand where me go (We go there, where the air blows!)
-		name = EVTOPTC_WCFW.10
+	option = {
+		name = EVTOPTC_WCFW_10 # Let the spirits guide us, we shall follow the winds! (either N or W)
 		piety = 50
 		random_list = {
 			50 = {
 				73 = {
 					province_event = {
-						id = WCFW.111 #Northern
+						id = WCFW.111 # North
 						days = 31
 					}
 				}
-				custom_tooltip = { text = EVTOPTA1_WCFW_10_tooltip } #we follow this path.
+				custom_tooltip = { text = EVTOPTA1_WCFW_10_tooltip } # and we travel towards the North.
 			}
 			50 = {
 				73 = {
 					province_event = {
-						id = WCFW.112 #Western
+						id = WCFW.130 # West
 						days = 31
 					}
 				}
-				custom_tooltip = { text = EVTOPTA2_WCFW_10_tooltip } #we follow another path.
+				custom_tooltip = { text = EVTOPTA2_WCFW_10_tooltip } # and we travel towards the West.
 			}
 		}
 	}
```

Comment:
To be honest, I think this option must be purged. Makes no sense. If you want random, just toss a coin.

Reply:
I'm of the same opinion, but since it was implemented before me I figured I'd push it in and see what happened in the review phase. I'll take care of this.

Reply:
Resolved in latest commit.